gbus/abstractions.go

```diff
@@ -18,6 +18,8 @@ const (
 	CMD Semantics = "cmd"
 	//EVT represenst a messge with event semantics in grabbit
 	EVT Semantics = "evt"
+	//REPLY represenst a messge with reply semantics in grabbit
+	REPLY Semantics = "reply"
```

Comment:
Are we sure that this is the correct place to add this sematic? since we are not using it anywhere except for the internal implementation of `setCorrelationId`....

Reply:
I know, but it is worth making it explicit IMO